### PR TITLE
adding 3rd party report to mail

### PR DESCRIPTION
**Pull request by adrima01**
Pull requests should be opened against the `main` branch. For more information on contributing to Lookyloo documentation, see the [Contributor Guidelines](https://www.lookyloo.eu/docs/main/contributor-guide.html).

## Type of change

**Description:**
Adding a 3rd party module section to the email template. 
We want to display if an URL is malicious or not according to the 3rd party modules to provide more information.

**Select the type of change(s) made in this pull request:**
- [ ] Bug fix *(non-breaking change which fixes an issue)*
- [x] New feature *(non-breaking change which adds functionality)*
- [ ] Documentation *(change or fix to documentation)*

---------------------------------------------------------------------------------------------------------

Fixes #issue-number

## Proposed changes 

*
*
*

### CODE REVIEW

**Comment by adrima01** [comment]
Maybe it's not a good idea to reuse the modules function in __init__.py to filter the response from the 3rd party modules? 
I tried to use it to avoid duplication

**Comment by AntoniaBK** [comment]
> I tried to use it to avoid duplication, maybe put the filter somewhere else?

I too would rather put all the filtering in an extra function that can be called when sending the mail and when displaying the 3rd-party lookups on the website.
So mail and website are not mixed together in one function.

**Comment by adrima01** [comment]
I think the concept of checking if the response is empty doesn't work.Urlscan triggers a scan (which wasn't the case at my lookyloo instance because it wasn't enabled)
So even if it isn't malicious, there will be something in the response

File: website/web/__init__.py [hunk 1]
```diff
@@ -817,7 +817,7 @@ def web_misp_push_view(tree_uuid: str) -> str | WerkzeugResponse | Response | No
 
 
 @app.route('/tree/<string:tree_uuid>/modules', methods=['GET'])
-def modules(tree_uuid: str) -> str | WerkzeugResponse | Response:
+def modules(tree_uuid: str, mail: bool = False) -> str | WerkzeugResponse | Response:
```
Comment (Rafiot) [inline website/web/__init__.py:820]: This method should only be used to render things on the UI, not in any other case. This is the expectation for all the methods with `@app.route`.

The new human readable response should be a dedicated method in lookyloo, or use the machine readable one (`lookyloo.get_modules_responses(tree_uuid)`), and make it human readable in `mail_send`.

File: website/web/genericapi.py [hunk 2]
```diff
@@ -389,7 +389,9 @@ class CaptureReport(Resource):  # type: ignore[misc]
     @api.param('comment', 'Description of the URL, will be given to the analyst.')  # type: ignore[misc]
     def post(self, capture_uuid: str) -> bool | dict[str, Any]:
         parameters: dict[str, Any] = request.get_json(force=True)
-        return lookyloo.send_mail(capture_uuid, parameters.get('email', ''), parameters.get('comment'))
+        from . import modules
```
Comment (Rafiot) [inline website/web/genericapi.py:392]: This is going to be very weird to debug and probably cause issues. Unless you have a very good reason (there are very few), you should never have to call `import` unless you're at the beginning of the file.

**Review by Rafiot — COMMENTED** [review]
Just a few notes

File: website/web/__init__.py (hunk 1 above)
Comment (adrima01) [reply website/web/__init__.py]: changed it now, hope that it works for every module (couldn't test every module).

**Pull request merged**